db/migrate/20180126144040_create_digest_run_subscribers.rb

```diff
@@ -0,0 +1,13 @@
+class CreateDigestRunSubscribers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :digest_run_subscribers do |t|
+      t.integer :digest_run_id, null: false
+      t.integer :subscriber_id, null: false
+      t.datetime :completed_at
+      t.timestamps
+    end
+
+    add_index :digest_run_subscribers, :digest_run_id
+    add_index :digest_run_subscribers, :subscriber_id
```

Comment:
Shall we have these as foreign_keys? and if so shall we have on_delete: cascades?

Reply:
Yep, updated.